build.gradle

```diff
@@ -41,7 +41,11 @@ repositories {
     }
     maven {
         name = "Cotton"
-        url = "http://server.bbkr.space:8081/artifactory/libs-release/"
+        url = "https://server.bbkr.space:8081/artifactory/libs-release/"
```

Comment:
this website doesn't utilize https, you're going to need to explicitly set `allowInsecureProtocol` to `true` instead.